dcs_deploy.py

```diff
@@ -412,20 +519,23 @@ def get_missing_resources(self, force_all_missing = False):
         for resouce in self.resource_paths:
             if os.path.isfile(self.resource_paths[resouce]) and force_all_missing == False:
                 continue
-            # return only resource which is possible to download
+            # Return only resource which is possible to download
             if(self.get_resource_url(resouce) != None):
                 res += [resouce]
         return res
 
     def download_resources(self):
         for missing_resource in self.get_missing_resources(force_all_missing = self.args.force):
+            if missing_resource == "rootfs" and self.args.rootfs is not None:
+                print("rootfs will not be downloaded, because you want to use custom rootfs.")
```

Comment:
At check here, that the path is viable.

Reply:
Added in 870ddbb54c200d29aa608aca14e37e03a1f89dba